data/filters/youtube-cleanup.yaml

```diff
@@ -8,6 +8,14 @@ params:
     description: Hide the text next to the dislike/share/download/report/save buttons below a video.
     type: checkbox
     default: false
+  - name: remove-comment-section
```

Comment:
nitpick: as the comment section is lower, I'd put that option as last (and the hashtags second), to keep the order consistent with the visual order on the youtube page

Reply:
See https://github.com/xvello/letsblockit/pull/153/commits/d55073975515b68f21a962c3c5c76810a5f0fa9e